src/System.Management.Automation/engine/parser/ast.cs

```diff
@@ -7889,6 +7895,31 @@ public MemberExpressionAst(IScriptExtent extent, ExpressionAst expression, Comma
             this.Member = member;
             SetParent(member);
             this.Static = @static;
+            this.GenericTypeArguments = new ReadOnlyCollection<ITypeName>(genericTypes ?? Array.Empty<ITypeName>());
```

Comment:
Let's do the following to keep consistent with how `arguments` are handled by `InvokeMemberExpressionAst`:
```
  if (genericTypes != null && genericTypes.Count > 0)
  {
      this.Arguments = new ReadOnlyCollection<ITypeName>(genericTypes);
  }
```

Reply:
Addressed